spec/system/publishers/publishers_can_invite_jobseekers_to_apply_for_vacancies_from_their_profile_spec.rb

```diff
@@ -11,7 +11,7 @@
            subjects: %w[english maths],
            working_patterns: %w[full_time])
   end
-  let!(:location_preference) { create(:job_preferences_location, name: "London", radius: 100, job_preferences: job_preferences) }
+  let!(:location_preference) { create(:job_preferences_location, name: "London", radius: 200, job_preferences: job_preferences) }
```

Comment:
Increased the radius to ensure this location showed up after changing Geocoder::DEFAULT_STUB_COORDINATES